scripts/changeScanner/index.js

```diff
@@ -14,28 +14,24 @@ const requiredChanges = ['CHANGELOG.md', packageFileName];
 
 const changedPackages = getChanges();
 
-const getChangedFilePath = ({ packageName, matchFile }) =>
-  changedPackages[packageName].find(changedFile =>
-    changedFile.includes(matchFile),
-  );
+const getChangedFilePath = ({ packageName, matchFile }) => {
+  const packageChanges = changedPackages[packageName];
+
+  return packageChanges
+    ? packageChanges.find(changedFile => changedFile.includes(matchFile))
+    : '';
```

Comment:
What has changed to make `packageChanges` sometimes `undefined`?

Reply:
this change https://github.com/bbc/psammead/pull/4368/files#diff-b91241351ff6aaef3a16d76228a999a9994e7038269586f10e5eb64e2b1d9657R45-R48

This is selecting the psammead root package which doesn't exist in `changedPackages` if there haven't been any changes in psammead root